code/modules/atmospherics/machinery/other/miner.dm

```diff
@@ -13,29 +12,31 @@
 	anchored = TRUE
 	density = FALSE
 	resistance_flags = INDESTRUCTIBLE|ACID_PROOF|FIRE_PROOF
-	var/spawn_id = null
+	var/list/gas_mixture = list()
```

Comment:
gas_mixture**s**

Reply:
It doesn't make sense for mixture to be plural.

The miner has one mixture, which has one or more gasses.